README.md

```diff
@@ -13,7 +13,27 @@ But the resources can be varied based on availability. However, make sure to
 use `--iso-url centos` part in above command as we have setup things on CentOS
 based minishift VM.
 
-Once the VM is ready, spin up a Jenkins server that can be used by the Jenkins
+Once the VM is ready, get into the vm using the command `$ minishift ssh`. Once you are in, apply the following steps to install an external registry. You are free to install it on a separate system, or your base machine, just ensure it is reachable from the minishift vm.
+
+    $ yum -y install docker-distribution
+
```

Comment:
> Once the VM is ready, get into the vm using

Once the VM is up, log into the VM using

Reply:
> apply the following steps to install an external registry

Why does the user need to install registry? No point mentioning "external" I guess. Maybe write "Docker distribution" instead?

Reply:
> You are free to install it

You can also install it

Reply:
> from the minishift vm

from the minishift VM